js/packages/components/chat/message/UserMessage.tsx

```diff
@@ -149,6 +149,7 @@ export const UserMessage: React.FC<{
 		cmd,
 	} = getUserMessageState(inte, members, convKind, previousMessage, nextMessage, border, color)
 
+	console.log(previousMessage?.medias?.[0]?.mimeType)
```

Comment:
I don't think this log was necessary, else code LGTM 👍

Reply:
Thanks I will fix it.